### ISSUE TITLE
Carousel is not working in android - form last commit

### ISSUE THREAD

**Issue by karthickvkumar** [machadogj/react-native-carousel-control#7]
Hi @machadogj, Recently i post a issue regarding initialPage props not working. After that you did a commit but now, Carousel is not working on Android, It simply scrolls like Horizontal scroll view. I even checked with v0.27 in android. still, im getting the same result.

![d14w3j](https://cloud.githubusercontent.com/assets/11032434/16169732/3970655a-3556-11e6-9fa5-156bf38a7a2a.gif)

Thanks in Advance!

**Comment by machadogj** [comment]
Hi @karthickvkumar, I tried with the latest and the following code seems to be working as expected:

```
          <Text style={styles.welcome}>
            Welcome to React Native!
          </Text>
          <Text style={styles.instructions}>
            See carousel below.
          </Text>
          <Carousel initialPage={1} pageStyle={ {backgroundColor: 'grey', padding: 20, height: 120} }>
              <Text>Hello</Text>
              <Text>World!</Text>
              <Text>From carousel</Text>
          </Carousel>
```

I'll keep doing some more testing, but let me know if that works for you.

**Comment by karthickvkumar** [comment]
@machadogj after wrapping `<Carousel>...<Carousel>` inside `<View>` with flex:1, its working fine! but its not setting the initialPage, Can you please put your working example for reference!

**Comment by machadogj** [comment]
@karthickvkumar what I pasted in my previous comment seems to be working on android. Which version of react-native-carousel-control do you have in your node_modules folder?

**Comment by machadogj** [comment]
@karthickvkumar turns out you were right on the initialPage bug. Don't know why it worked for me, but it should be fixed in version 1.0.3 (https://github.com/machadogj/react-native-carousel-control/commit/fc864259cfe81a1409ba985c877c4b206abde822).
Thanks for reporting it!
